keps/sig-storage/1979-object-storage-support/README.md

```diff
@@ -243,147 +242,161 @@ A `Bucket` is not deleted if it is bound to a `BucketRequest`.
-1. `accessGranted`: if true access has been granted to the bucket. If false then access to the bucket has not been granted.
+10. `parameters`:  A map of string:string key values.  Allows admins to control user and access provisioning by setting provisioner key-values. Copied from `BucketAccessClass`. 
+11. `accessGranted`: if true access has been granted to the bucket. If false then access to the bucket has not been granted.
+12. `credentials`: a `SecretReference` to the sidecar-generated Secret containing access credentials. This secret exists in the provisioner’s namespace, is read by the cosi-node-adapter, and written to the secret mount defined in the app pod's `csi-driver spec`.
+13. `accountID`: username/access-key for the object storage provider that identifies the user with access to this bucket. This value is minted by the driver when the `BucketAccessRequest` omits a ServiceAccount.
 
 #### BucketAccessClass
 
-An immutable, cluster-scoped, custom resource providing a way for admins to specify policies that may be used to access buckets. A `BucketAccessClass` (BAC) is required for both greenfield and brownfield use cases. Besides naming the provisioner, a BAC references a config map which is expected to define the access policy for a given provider. It is typical for these config maps to reside in each provisioner's namespace, though this is not required.  Unlike BucketClasses, there is no default BAC.
+An immutable, cluster-scoped, custom resource providing a way for admins to specify policies used to access buckets. A `BucketAccessClass` (BAC) is referenced by a user [BucketAccessRequest](#bucketaccessrequest) and is used to populate the [BucketAccess](#bucketaccess) instance. A BAC references a config map which is expected to define the access policy for a given provider. It is typical for these config maps to reside in each provisioner's namespace, though this is not required. Unlike a [BucketClass](#bucketclass), there is no default BAC.
 
 ```yaml
-apiVersion: cosi.io/v1alpha1
+apiVersion: objectstorage.k8s.io/v1alpha1
 kind: BucketAccessClass
 metadata: 
   name:
-provisioner: [1]
-policyActionsConfigMap: [2]
+policyActionsConfigMap: [1]
   name:
   namespace:
-parameters: [3]
+parameters: [2]
 ```
 
-1. `provisioner`: (optional) the name of the driver that `BucketAccess` instances should be managed by. If empty then there is no driver/sidecar, thus the admin had to create the `Bucket` and `BucketAccess` instances.
 1. `policyActionsConfigMap`: (required) a reference to a ConfigMap that contains a set of provisioner/platform-defined policy actions for bucket access. It is seen as typical that this config map's namespace is the same as for the provisioner. In othe words, a namespace that has locked down RBAC rules or prevent modification of this config map.
```

Comment:
The user experience with config maps is better because the access specification format for different protocols differ, and config map data is just much more readable as compared to say, an escaped string or base64 encoded field.

Reply:
Why is it better to pop out to a cConfigMap than to just include a list of values here?  They are both YAML in the end, but  now the user has to create ANOTHER object (and clean it up one day) and the poor slob who has to figure out how your cluster works after you quit gets to bounce around between types.

Also a configmap is a map - this doesn't seem to need the keys at all?  `[]string` seems more appropriate to me?